packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -131,7 +137,7 @@ class MethodChannelCamera extends CameraPlatform {
     );
 
     if (_channels.containsKey(cameraId)) {
-      _channels[cameraId].setMethodCallHandler(null);
+      _channels[cameraId]!.setMethodCallHandler(null);
```

Comment:
nit: My preference is that `!` should only be used if you can guarantee that something is non-null by looking within the same scope, otherwise it should be handled. But, I also just started using null safety, so my intuition may be wrong on that.

Also, the `MethodCall` is still invoked earlier in the method, but a `null` exception can still be thrown after. This could probably be checked and handled before the `MethodCall`.

Reply:
I had to think a bit about this, but I agree. If possible it makes sense to only use the `!` within the same scope and where access is clear. Thanks for pointing this out. I have updated the code.